.vscode/extensions.json

```diff
@@ -12,6 +12,9 @@
     "kortina.vscode-markdown-notes",
 
     // Image-pasting for markdown
-    "mushan.vscode-paste-image"
+    "mushan.vscode-paste-image",
+
+    // Spell checking for text, markdown and latex
+    "ban.spellright",
```

Comment:
@riccardoferretti why Spellright and not some other extension like [Code Spell Checker](https://marketplace.visualstudio.com/search?term=Code%20Spell%20Checker&target=VSCode&category=All%20categories&sortBy=Relevance)?

Code Spell Checker has 5.3 million downloads compared to Spellright's 300,000?

The reason I ask is I have this issue with Spellright https://github.com/bartosz-antosik/vscode-spellright/issues/210 when using it for other projects in other programming languages.

Reply:
This is just what I was using at the time, but per https://github.com/foambubble/foam/issues/1068 we have been thinking of removing it from the list of recommended extensions

Reply:
Thanks for the reply @riccardoferretti 👍